test/ecdsatest.h

```diff
@@ -0,0 +1,10216 @@
+/*
+ * Copyright 2019 The OpenSSL Project Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License 2.0 (the "License").  You may not use
+ * this file except in compliance with the License.  You can obtain a copy
+ * in the file LICENSE in the source distribution or at
+ * https://www.openssl.org/source/license.html
+ */
+
+#ifndef ECDSATEST_CAVS_H
+#define ECDSATEST_CAVS_H
+
+/*-
+ * NIST CAVP ECDSA KATs:
+ * https://csrc.nist.gov/CSRC/media/Projects
+ * /Cryptographic-Algorithm-Validation-Program
+ * /documents/dss/186-3ecdsatestvectors.zip
```

Comment:
nit: I believe the URL should stay in one line, even if it defies the style guidelines